### DISCUSSION TITLE
Voting power (Who should vote and how to distinguish which amount of power a voter has?)

### DISCUSSION THREAD

**Discussion by Phyloiota** [iota-community/Community-Governance#7] [Ideas]
Probably the most difficult Question of all... How can we find a way to make the vote as fair and balanced as possible?
For reference: Token distribution over addresses as of 21.04.2021:
https://docs.google.com/spreadsheets/d/1x1jOIrEe2-gGE22HyxRTyE8p6ABZmytpvTvSYksSS6k/edit#gid=0
minus 65 TI that rest now in Treasury.

**Comment by Cryptec** [comment]
I have thought a white now about this topic. And as i said previously  is in my opinion the most important point of such a Voting Mechanism is to have it truly decentralized. Dont sure if its wise to implement such a Mechanism to for Profit third Parties like discord. Also as others said i Think the right step is one Person one vote. With more Platforms involved lts easy to harm this System. In my opinion the First step should be to signalize a vote over the nodesoftware. And maybe a did can be transferred in a second step to the wallet.

Let me explain my thoughts behind this: we absolutelly should Not force people to use third parties like discord, and the only way to absolutelly, proove someone deserves a vote, without some external requirements or the possibillity to  harm is to run the core protocol.

Of course you have to inplement additional security steps like a node have to run At least one Month or so.

(Copied from my discord conversation in #governence-discussion)

Also from my point of experience, it is absolutelly impossible to prevent people from running Multiple accounts/ wallets/ nodes to get Multiple votes. So the initial hurdle must have some strength to prevent people from harming the System with Multiple votes. (Like the requirement of running a node for some time first)

**Comment by datahomelab** [comment]
I have been reading all the material that has been posted in #ideas-suggestions in the last couple of days, and it is not only promising, but also quite impressive in quality terms. I was thinking that, there is a pressing matter that has to be addressed first, that is, the initial vote regarding the Community Fund. A solution for this initial vote has to be implemented in the short term, that might not be able to cover all the angles that are being discussed regarding governance.

I think the priority should be to establish the set of parameters that have to be considered for this crucial, and initial vote. Getting this right is of paramount importance, cause it will be the event upon where any governance system will be built upon. If the foundations of this building are not stable, the governance system can be criticized for the years to come.

Essentially, the question is: How can we as a community come up with a fair voting system that can be implemented in the short term to cater for this important decision, and that will help settle that situation once and for all?

Let's start from the most basic principle regarding voting, and build from there: One person, One vote. That is the most basic and philosophical assumption about a voting system... Is it possible to start building from there, as a first principle?

(And this is a question that I have been meditating upon for a few days now, which might be right, or wrong...)

**Comment by thatlur** [comment]
For me the most important part about designing a good governance system is that you don't have one particular type of user have control and instead the power is distributed amongst a variety of different types of people who may all have different reasons for using iota. Just because you don't have iota tokens does not mean you shouldn't get a say in the future of iota.

For example you may run a project that relies on iota's data transmission and not have any tokens
You may run a node without having any tokens.
You may be a developer or prolific community member without any tokens.
Someone might have a project built on smart contracts without having any iota tokens.

How do we make sure these people are able to have a say? These people are still invested into the ecosystem even without holding iota tokens.

My idea is to come up with a number of parties that are invested in the ecosystem in some way and split the vote between them.
A vote token will be minted for each vote and distributed to each party.

I have thought of 5 different parties that will be invested in iota that should get votes, there are probably more I haven't thought of as well.

1. whales - They shouldn't have all the power but they have the most skin in the game and so should get a say. Vote tokens should be given per iota held.
2. active users - People who are actually using the network for day to day transactions should get a say in the network. Vote tokens to be given if you have made a transaction in the 3 months before a vote is announced.
3. node runners - node runners are the backbone of the network and deserve a say. Vote tokens to be given if you run a node.
4. projects built on iota - once smart contracts are released and people start building projects on iota I imagine more people will be interacting with iota through these projects than directly with the tangle itself. The people running these projects and potentially their users should have a say. (not sure on the best way to give out these vote tokens)
5. developers and highly active community members. If we have a digital identity linked to discord/github/reddit/twitter you could receive vote tokens if you are active in these communities.

Each party will get an equal share of the vote. This means that whales will have 20% of the vote, active users 20%, etc. For something to get through it will need to convince multiple types of people to vote for it.

**Comment by ricardroyo** [comment]
Looking at the token distribution and assuming that we set a limit of 100Gi as Dom suggested in the #governance-discussion that would leave as in a situation where 83.23% of the IOTA supply (owned 1319 addresses) would be subject to game the system by splitting the tokens. 
Even though the numbers might seem alarming, I don't think this should be a very likely scenario since most of these addresses are cold wallets (from exchanges) or stored in custody providers. In order to cheat the system, they would have to split them in multiple addresses and transfer them to firefly (I guess??)

One option that we could consider is to have some kind of software that would allow us to monitor these addresses to see if they participate in the votes and if they split the tokens. That would allow us to see if the system is being gamed and give us time to mitigate the issue.

**Reply by WernerderChamp** [reply]
I guess that would work, but still, how are we going to mitigate if it is happening? And I guess a lot of users already have their holdings split over multiple addresses, that might also apply for whales

**Reply by Deep-Sea-888** [reply]
I think this is a great option for the token holder variable. I also think, at least for this initial vote, we really need to look at "what are the benefits from being gamed". So for an exchange for instance. A major exchange with hundreds of Ti's that could substantially change the vote, would it be worth them to go to their custody holder, use their multi-party signing protocol to put the tokens into firefly, program a bot to make thousands of profiles, and then vote for what? To burn, to AirDrop, to deposit into one of their own addresses"? The whole process, being decentralized and on the Tangle will be visible. It will be easy to see such deception and pinpoint the culprit. So the question remains, would an exchange go through all that and risk their billions of crypto, and hundreds of millions of annual profit simply for $100 million worth of IOTA? Which, once the deception was found, the price of the token would drop, thus devaluing the stolen funds." 

I personally think, again, just for the token holder variable, having a 10 Gi or 100 Gi cap, and using Firefly, mitigates the risk. 

To further mitigate manipulation risks the voting can be diversified among parties as stated above. Such as user interaction, node owners, etc. 

Given the options to be voted upon will look something like AirDrop, Burn, DAO, Leave them as is... not one of these options benefits that much more than another which would incite whale manipulation. A whale might see short-term value with a Burn, but another whale might see long-term value in a DAO (all in regards to price and value of their stack). 

Again, this is just for the initial vote. If the DAO is voted then a more in-depth and more protective structure needs to be voted on so an entity, group, or whale can't simply vote to send the funds to their wallet address. Different discussion and situation from our current vote I think.

**Reply by ricardroyo** [reply]
Yes, I do think the economical incentives to try to game the vote are small for exchanges and big players, but we should not forget that IOTA has had many enemies over the years and this might be seen as an opportunity to harm the community.

Somebody in the IOTA Discord suggested using the "token holding periods"  as a way to limit the possibility of whales to game the vote. My understanding how this could work is to take a look at the milestone that confirmed a certain output to weight how much voting power your tokens give you. If you've been not moving your tokens for a long time (old milestone confirming your output) then you get the full "voting power". If you've recently moved them, you get less power. 

This is just a rough idea and I don't really know what are the implications or how difficult this would be, but I'm interested to know what you guys think.

**Reply by ACC30** [reply]
> Yes, I do think the economical incentives to try to game the vote are small for exchanges and big players, but we should not forget that IOTA has had many enemies over the years and this might be seen as an opportunity to harm the community.
> 
> Somebody in the IOTA Discord suggested using the "token holding periods" as a way to limit the possibility of whales to game the vote. My understanding how this could work is to take a look at the milestone that confirmed a certain output to weight how much voting power your tokens give you. If you've been not moving your tokens for a long time (old milestone confirming your output) then you get the full "voting power". If you've recently moved them, you get less power.
> 
> This is just a rough idea and I don't really know what are the implications or how difficult this would be, but I'm interested to know what you guys think.

it seems to me that how we moved the coins yesterday is less of a character for us than how we plan to move them tomorrow.  as I wrote a stronger tool for freezing after voting.  this will strongly show our intentions to stay in the project, and not just get quick money.

**Reply by ebk1960** [reply]
I link the idea of a dynamic voting structure based on how long you've held your tokens.

**Comment by ACC30** [comment]
it seems to me that it is necessary to separate the votes according to their scale.  mechanism for voting to burn or form a DAO and "who won the most original arduino project in the competition."  In the first case, I see no reason to make efforts of those interested in artificially winding up votes in one direction or another.  for this, it will be enough to add colored coins, and whether there will be votes from one wallet or from 1000 does not matter, colored coins will be blocked for some time and everyone will decide for himself how important it is for him to vote or leave funds in circulation.  and the voice?  1 iota is one vote.  I think it will be fair.

**Reply by ebk1960** [reply]
I don't think we're suggesting coloured coins would be used for voting. However, the projects behind those tokens could possibly have rights to vote as they are building out the IOTA ecosystem.

**Comment by dudeamir** [comment]
As the vote going to be in June, the complexity of the voting mechanism should be minimised. It is necessary to separate the solution used for June Vote on Treasury and future DAO voting systems. For DAO based on all models available and other valid arguments made here and in discord, should be discussed. However, for June Voting, as it is not much time left to debate to come to the perfect mechanism, I suggest 1 IOTA, 1 Vote, with a cap! (10G-100G or higher as suggested by others). 
However once DAO forms, different stake holders should be considered in a fair mechanism. Further development on IOTA tech would eventually enable all those features, therefore DAO should be agile and evolving. Starting with a model for voting and keep developing and making it better. Coming with a fixed solution at beginning, oversees loop holes and drawbacks. Therefore, the DAO should be capable of evolving itself. Therefore a mechanism to how enabling these ongoing changes to Voting Power is something to be discussed as well. In my opinion, starting with 1 IOTA, 1 Vote, the DAO can enable future votes on adding more mechanism to the Voting Power.  
Also, in my opinion, to put the first ever vote of IOTA on the treasury with an untested mechanism is too much to begin with. An initial dummy vote would clean the pipe-line, show if there is any unknown variable in the process. Therefore, I would as mentioned in #governance-discusstion, suggest a dummy vote, a week prior to the main vote. This dummy vote can be on sth not as important as treasury but somehow funny and stupid to engage the community to vote on it. Sth simple, sth not that serious! It is obvious that IOTA community is different than Doge community, but as it becomes more clear than younger people share what is funny, this brings also an opportunity for IOTA to make it also an indirect marketing. Bringing some stupid fun things into publicity of IOTA doesnt make it any less serious in terms of the technology.

**Reply by ACC30** [reply]
tell me, why do you propose a limit on the minimum number of coins?  do you think that a member with 100 Miota is less interested in development than a member with 100Giota?  Our dedication to iota is not determined by the size of the cough.  Small devoted wallets can pay for the votes with their time by showing loyalty.  it would be nice to discuss the following idea.  A whale with 10 Ti wants to burn coins.  He hopes for a rapid rise in price due to this and quick enrichment when selling at the peak.  100 intuzaast devotees and sincerely believing in the future have 1G each.  If we can make a mechanism whereby, say, each week of blocking coins for moving over the first one gives an increase in the weight of votes.  in general, the temporary blocking mechanism can help you to be much more attentive to your voices and to manage your voices in a more balanced way.  so I'll return to the example.  10 Ti gave all his votes for burning (10 Ti votes), but 100 people can defeat him.  they "invest" their coins in votes and block coins for 2 years (showing loyalty to the coin).  every week is a multiplier.  103 weeks × 100 people × 1 Giota = 10.3 Ti votes.  The decision to create a DAO won.  This mechanism will also discourage the desire to participate in the voting of speculators and stock exchanges, who, by and large, do not care what the fate of the coin is, they earn here and now on its movements.  The freeze is not beneficial to them, which means that their votes will remain unused, which in turn means that only truly loyal participants will make a significant contribution to the voting results.

**Reply by ACC30** [reply]
Cough = wallet )

**Reply by dudeamir** [reply]
In my head the cap was on the maximum, not on the minimum. Absolutely even 1iota should be able to vote, removing holders lower than 100G doesnt make sense to include such large group of holders. Taking a closer look at the total supply distribution: https://docs.google.com/spreadsheets/d/1x1jOIrEe2-gGE22HyxRTyE8p6ABZmytpvTvSYksSS6k/edit#gid=0 
I personally prefer a 1Ti or 10Ti as the maximum to remove in terms of the document "Queen Bees" to stir the vote to their favour by large margin. This would be a debate as well, where to put the maximum! Or does it need a maximum? If the requirement would be to have it in firefly, it would at-least remove all exchanges. Then, a no cap what so over would work as well. 
Anyway, I am against removing voting power for smaller holdings!

**Reply by Deep-Sea-888** [reply]
Certainly not less interested, but with the 10 Gi / 100 Gi cap, this only limits whales voting one way or the other and dominating the vote. To me though, realistically, neither benefit a whale one way or another. 65 Ti is about 2.5% (If I calculated) correctly of the total supply. So the jump in price is 3%??? Compare that to a normal day 5% to 20% move either way. But more importantly, compare that to 65 Ti providing development and furthering adoption which ultimately raises the price way more than 2.5%.

Saying all that, putting a cap doesn't make it impossible for someone to code a bot to create thousands of Firefly profiles should someone want to do that, but is it really worth it? Personally, I would not risk losing keys or passwords on a thousand profiles simply to sway the vote either to burning or DAO treasury.

It just seems, making an assumption, that Firefly can allow a vote come June, based on token distribution, and some sort of cap can be placed on it. Ultimately, for the "initial" vote, this seems the easiest and best approach. "If" the DAO gets voted in then for sure a completely in-depth proposal/voting structure would need to be built.

**Reply by dudeamir** [reply]
Agree, upon agreeing on DAO, then all these details and mechanism should be discussed and voted. For June vote, given the possibility of distributing tokens to multiple addresses to pass the max cap on tokens, and complexity of building a mechanism to prevent it, easiest and most safe with least controversy is No Cap, one iota, one vote.

**Comment by Phyloiota** [comment]
Just to keep it here also as we had a talk about that today and Stefano did put the great idea up to use the Node ID as a voting mechanism. So we could give Node operators a voice in the vote issuing them a IOTA DID that verifies the unique Node ID.
We could then decide how much "power" Node owners should have compared to token owners....
A 50/50 split of the voting power could lead to something like this:
Total supply is 
2,779,530,283,277,761 Iota (2,779 Pi)
remove the ~63 TI from the treasury wallet
= 2.716.530.283.277.761 votes for token holders and 2.716.530.283.277.761 votes for Node operators

As we dont know how much nodes are operated we could do a good guess and say we have 2000 Nodes in the Network....
This would give 1 node the same voting power as a Token holding of 1,358 Ti .....

Something to think about....

**Reply by ACC30** [reply]
create and configure node $ 30 per month.  buy 1.3 Tiota much more)

**Reply by basvansambeek** [reply]
Exactly. While a node ID itself could be secured, we have an arbitrary division of the vote and the ability to set up dozens of not hundreds of nodes. And how likely is it that this will be abused/tested? Ask muxxer. He’ll tell you.

You’d have to weight it’s vote according to something like age to verify some sort of genuine contribution to the network, but also uptime, and synchronization to close the door on abuse.

And then you suddenly have an advanced and untested reputation system that is not trivial to monitor in a decentralized way. Let alone build it.

**Reply by ACC30** [reply]
after launching chrysalis and updates, were the nodes restarted?  I am not saying that the mechanism of trusting the nodes cannot be implemented.  just if we are talking about voting in June or July, we need to estimate very well how many votes will be given to the nodes.

**Reply by Phyloiota** [reply]
We can use uptime of Nodes in the decision - so lets say only nodes that started in the first 2 weeks since the chrysalis upgrade and run since them could vote 
![image](https://user-images.githubusercontent.com/77154511/117895981-222aeb80-b2f2-11eb-99f8-3d357b6f1a7a.png)
needs to be figured out how to make this metric tamperproof

**Reply by WernerderChamp** [reply]
@Phyloiota I think this is a bad idea, because you have to restart your node to change some config parameters like whilelisted API routes. Or your node crashed due to a bug.
You can run multiple hornet nodes on a server or modify the code to mimic being multiple nodes (e.g. with proxy servers). The hardware costs are not enough to protect this system from sybil attacks

**Reply by Phyloiota** [reply]
> @Phyloiota I think this is a bad idea, because you have to restart your node to change some config parameters like whilelisted API routes. Or your node crashed due to a bug.
> You can run multiple hornet nodes on a server or modify the code to mimic being multiple nodes (e.g. with proxy servers). The hardware costs are not enough to protect this system from sybil attacks

Yeah Werner - muxxer already explained that this Node id voting will not be a bulletproof thing so the idea to use it will not be suitable. We will need to wait until we have Mana in the Network before we can give a proper vote to Node owners based on Reputation / Consensus Mana etc etc...
So it seems the initial "burn or use" vote will be purely based on token holdings...

**Comment by Phyloiota** [comment]
For a later established governance MANA is a wonderful way of establishing voting power and also a great and easy way for token holders to cast their vote.
As holders can freely decide to which node they can pledge their access Mana a simple pledge operation towards a node could represent a vote.
And consensus Mana gained by valuable Nodes for participating in the Network could be the metric for Node operators to figure out how many power they would have in a vote...

**Reply by dudeamir** [reply]
I think in one case, this can cause a controversy. I thought MANA would enable token holders to earn as pledging their MANA for certain nodes to enable a higher bandwidth and be incentivised to do so. Then, not only they get paid to pledge, but also their pledge is used as a voting power. In this case, to game the voting, a node can transiently provide a high incentive for pledging to gain a higher voting power. This case is only with assumption that there is any incentives in pledging tokens for holders. Not sure about it, but I guess I heard in one of weekly videos.

**Reply by basvansambeek** [reply]
I was one of the people who suggested it, and before using it, I would discuss this with the IOTA Foundation research department, because it would create an incentive for a consensus mana market, opening an attack vector now deemed unlikely. But since there are already two mana, I suggested a third mana for community votes might be an option.

There are still a lot of things that need to be solved then, like initial distribution, default setting of node ID's in various wallets, etc., but at least we have an infrastructure intrinsic to the system, that enables proportional representation and can mimic a liquid democracy with votes to be delegated to somebody that champions your ideals when you don't want to vote for some reason.

**Comment by Phyloiota** [comment]
copied from discord:

Phylo [Community DAO - lets go!]
 — 
heute um 07:58 Uhr
So as Dom yesterday said - IF is developing a function in Firefly for a weighted voting (based on token holdings imho). So this will be a first technical solution to make voting possible and if its done by IF it will be pretty much tamperproof. 
Using Firefly will have a few advantages: It will very much limit the power of exchanges. I absolutely think that not one serious exchange will participate in a voting using Firefly. Exchanges have different Wallets for their stake: a hot wallet that represent the coins currently actively traded in the orderbook (+ x Amount to cover orders of fresh market partcipants) and one or several cold storages.
The hot wallet coins cannot be used to vote because for this they would need to be transfered to firefly and the liquidity of the market would be directly gone.
The cold wallets are hardware storages that are usually safeguarded and not touched in normal operations.
Now think about - an exchange installing Firefly on their system, opening their millions of dollars cold wallets - transferring them into firefly to cast a vote? Never ever gonna happen. Firefly is an untested software and doing this would be a huge security risk for them - no serious Manager of the cold storage department would ever take the risk to do that and explain later why he did put the holdings and capital into such a risk.
Next good point is: It will create a big outflow from the exchanges of IOTA holders that want to participate in the vote - and therefore reduce the liquidity on exchanges that again usually will lead to a price increase in general.
IOTA holders that dont care about the project will anyway leave their coins on exchanges - they will not even know that a voting is up because they dont follow the project closely - so the holdings of those are out of the game. Others that dont leave it on exchanges but dont care have it maybe since years on a ledger... maybe will not even transfer the coins when ledger update is ready - because again they dont care and check only every 6 months the IOTA project and only wait for a nice price one day to sell...
That leaves the private big whales that are still actively following the project. They mostly have bought early and have a few hundred Gi or even a few TI from the early days... if they have been following closely until now and decide to vote now there is nothing to say against it - they have been part of the community all the time, didnt sell thougout all the drama that happened, kept their holding through the endless bear market. They definetly earned also a saying in the vote.

So we could say then for the initial vote we rely only on this, or try to put a second layer in it to reduce the influence of pure stake: 
give the Nodes a possibilty to vote also. Hornet devs could develop a plugin for the nodes that makes a vote possible based on: Node ID and uptime of the Node.
In this way we could give all nodes that started the network lets say in the first 2 weeks and are online and running since then also a vote. It would be upon us to figure out how much power to give to the Nodes in the voting mechanism compared to the power of the pure stake.
And Node operators will be the one that have to put a decision in place - they are running IOTA - they keep the network alive - without them IOTA does not even exist.
So giving them a chance to have a saying is more then fair imho, because the invest money and time in the project and need to be taken into account and valued as community also.

**Reply by Phyloiota** [reply]
To add to this - IOTAS daily trading volume over the last months is constantly around 200 - 400 million US - Dollar on the Spot markets - so this will be another 100 - 200 TI that will not participate in any vote as its pure coins of traders changing hands...
Adding an amount of coins locked in Future and margin contracts on exchanges....

**Reply by dudeamir** [reply]
It makes all good sense, its now a matter of the feasibility of a mechanism that can bring node operators to the vote as well till end of June! 
Once governance is voted, i guess community coin, participation coin or any name, with some form of distribution mechanism to include all these stakeholders (node operators, community contributors, ...) can be though. If a safe and easy way to include node operators can't be integrated till end of June, or earlier to test it, I guess then should be kept for including after the initial vote.

**Reply by Phyloiota** [reply]
Regarding the Node ID voting here the opinion of Muxxer from discord regarding it:

"muXxer
 — 
heute um 08:15 Uhr
Good idea, but I think this is not possible.
This "node identity thingy" is easily gameable without mana IMHO.
I wouldn't add that second layer of influence tbh
Do it based on stake, and you will be fine.
Popov also had the idea to have a maximum cap for the voting weight per person.
So big token holders would need to split up their coins to have their full voting weight, still possible, but it makes it a bit harder for them."

**Reply by sascha1337** [reply]
+1 for muxxer, that would make it a little bit more diverse.
and maybe value those maxcap votes 0.8 weight, instead 1to1 - maybe.

**Reply by basvansambeek** [reply]
I’m happy to see mathematicians and devs chiming in. It will certainly help build trust in the outcome of the vote

**Comment by rophiroth** [comment]
I see all with a lot of utopic view. We are talking about voting power about iota business, like our economical system that the one who have more, have more power... that's the doom of our civilization, we must transcend that kind of distribution, if it's not 1 person 1 vote we are doomed, because now we could be using iota to vote simple iota things, but in the future could be global political decisions; imagine an utopic world where people are choosing how spent resources (money, hour of men, etc)  for the good of everyone instead of accumulation, we could even transcend the necessity of money and live in fully abundance on every corner of the planet.

**Reply by ACC30** [reply]
we do not assume that the DAO will make political decisions.  for political voting, people must first show their face and associate it with their address.  it's different.  it's like green passports for covid19 vaccination

**Reply by ACC30** [reply]
today there is no way to identify 100 addresses from one person or from 100 people.  in the distant future, I believe the person can be identified by DNA.  and even so, nothing prevents you from making some seeds for your sister, brother, mom and dog).  and if the dog is also a Shibo Uno breed, then buy it a Tesla)))

**Reply by Phyloiota** [reply]
Agree on your wishing, but for the current upcoming vote that might happen in around 6 Weeks there is no technical solution to establish a 1 person 1 vote decision. 
And even then - we come to the other questions regarding that topic..... 
We have a lot of people on this planet. Theoretically they all should be able to vote? I dont think you want this... 
So; lets say -  all people that "have something to do with IOTA" should vote? Define it... 
All people that own IOTA? How much IOTA? 1 Mi, 100 Mi, 1 Gi? define it? 
1 have 1000 Mi and split it to the 1000 people of my community or my village or to 1000 random seeds that i all control? Proof that i do something wrong and there are really 1000 people or just me splitting my identity please ...
Everyone that is part of the IOTA community should vote? ... Again - define it and show a technical solutions that guarantees that it is not gameable....

This is the very very difficult discussion that we are doing here... Who sets the rules and who defines who is a voter?
And again without a technical solution that executes this vote on the tangle and cannot be tricked we will not be able to do this....

If we look into other projects that run DAO all had the same problem and the same questions in the beginning - to define how much value the invested money should have in a vote! 
And all of them have come to the solution so far to have a highly token holding based power distribution. 
They try to balance it with locked stake periods, so if you hold only a low amount of coins, but are okay to leave it for a long time locked in a "voting decision", you can gain more power in this vote then someone that deposits a large amount of funds only for a short time... this is something we can think about.... And it would also reduce the incentive to "buy votes" ...

But i have not seen a single project yet that completely decouples your personal investment (money) from your ability to make decisions in the project....
Only of course, if you give the power totally to a council of elected people that do all the decision behind closed doors (still the election of those people is based on stake usually), but this is the opposite of a decentralized Organisation and liquid democracy.

What i think we can achieve with IOTAs technology is more than other platforms can do. As soon as IOTA 2.0 is running, we have an amazing digital Identity solution, so we can think about a way of creating this DID's for voting and verifying them somehow.
We can create single voting tokens linked to DID's that can be used to deposit in votes and returned to the owner after the vote, this can all be executed by smart contracts. We can think of giving out extra vote for achievements in the community or the ecosystem. But we can never take away the power totally from people that own the tokens... it is also not fair to do this.
But a DAO can balance all those factors based on rules that everyone had agreed on, and those rules can also be changed again if needed by the community whenever a majority feels it needs to be changed

Bringing up for discussion is important to give the IOTA community an idea and a vision of what will be possible to achieve and to build with a treasury and how it could work. 
But for the first vote we seem to be bound by staked voting using firefly wallet.
This will, if the vote is pro DOA also be a legitimate proof of the opinion of "IOTA" and therefore give a governance the ability to start build a system. The development of a voting system that is not possible to influence or trick in any way will be the first challenge on the way to establish that true decentralized democracy...

**Comment by rophiroth** [comment]
Having iota could be used, but never as power of vote, but could be used to
have more participation on validation voter ID. Maybe as the tangle works,
we should work to validate our identities, and people could validate their
human participation by owning iota and/or developing and/or forum
participation and/or validating others documents, a quick videocall, or
anything. There will be always possibilities of cheating, specially if you
plan to vote soon... so all real solution are long term, all possible
efforts for doing 1 person 1 vote will help in all sort of votation in the
future.

Thank you

**Comment by mikestaub** [comment]
Perhaps we could use this to validate uniqueness? https://www.brightid.org/
